examples/ct/ct_server/main.go

```diff
@@ -34,26 +36,49 @@ import (
 	"google.golang.org/grpc/naming"
 )
 
-// Global flags that affect all log instances.
-var (
-	serverHostFlag    = flag.String("host", "localhost", "Address to serve CT log requests on")
-	serverPortFlag    = flag.Int("port", 6962, "Port to serve CT log requests on")
-	rpcBackendFlag    = flag.String("log_rpc_server", "localhost:8090", "Backend specification; comma-separated list or etcd service name (if --etcd_servers specified)")
-	rpcDeadlineFlag   = flag.Duration("rpc_deadline", time.Second*10, "Deadline for backend RPC requests")
-	logConfigFlag     = flag.String("log_config", "", "File holding log config in JSON")
-	maxGetEntriesFlag = flag.Int64("maxGetEntriesAllowed", 0, "Max number of entries we allow in a get-entries request (default 50)")
-	etcdServers       = flag.String("etcd_servers", "", "A comma-separated list of etcd servers")
-)
+type config struct {
+	Host                 string
```

Comment:
The Golang JSON package [requires that fields be public if you want to unmarshal into them](https://godoc.org/encoding/json#Unmarshal).